metapool/sample_sheet.py

```diff
@@ -1638,8 +1653,9 @@ def __init__(self, path=None):
 
 
 def _parse_header(fp):
+    # https://stackoverflow.com/a/43586874
```

Comment:
During some testing in qiita-rc I saw this line fail with the `Error tokenizing data. C error:`, and the link suggest forcing the engine solved the issue.